packages/components/header/headerAutoComplete.js

```diff
@@ -1,5 +1,13 @@
 import AutoComplete from './autoCompleteConfig';
 
+// Check if search URLs are set as globals on window object
+const overwriteSearchApiUrl = window.NHSUK_SETTINGS && window.NHSUK_SETTINGS.SUGGESTIONS_TEST_HOST;
+const overwriteSearchPageUrl = window.NHSUK_SETTINGS && window.NHSUK_SETTINGS.SUGGESTIONS_TEST_HOST;
```

Comment:
`overwriteSearchPageUrl` should look for `SEARCH_TEST_HOST` not `SUGGESTIONS_TEST_HOST`.

To avoid duplication, we could rewrite like this I think (I haven't tested it):
```js

const searchApiUrl = (window.NHSUK_SETTINGS && window.NHSUK_SETTINGS.SUGGESTIONS_TEST_HOST) ||  'https://www.nhs.uk/s/suggest.json';
const searchPageUrl = (window.NHSUK_SETTINGS && window.NHSUK_SETTINGS.SEARCH_TEST_HOST) || 'https://www.nhs.uk/search/';
```